Sources/HummingbirdAuth/Sessions/SessionStorage.swift

```diff
@@ -125,6 +123,23 @@ public struct SessionStorage<SessionType: Codable>: Sendable {
         )
     }
 
+    /// update existing session and return cookie if expiration date is set
+    ///
+    /// If session does not exist then a `sessionDoesNotExist` error will be thrown
+    public func updateAndReturnCookie(session: SessionType, expiresIn: Duration?, request: Request) async throws -> Cookie? {
```

Comment:
```suggestion
    public func updateAndCreateCookie(session: SessionType, expiresIn: Duration?, request: Request) async throws -> Cookie? {
```

This function signature is quite awkward. We should come up with a better name, or even consider keeping it internal for now

Reply:
I changed the name and made it internal